lib/interfaces/include/LTCSPIInterface.tpp

```diff
@@ -37,7 +40,7 @@ void write_registers_command(int cs, std::array<uint8_t, 4> cmd_and_pec, const s
 
     _transfer_SPI_data<4>(cmd_and_pec);
     
-    SPI.transfer16(0); // need to do this # of times devices
+    //SPI.transfer16(0); // need to do this # of times devices
```

Comment:
Why are these commented? Was it ultimately unnecessary, or will they be added back in at some point?